src/main/java/org/broadinstitute/dsm/model/elasticsearch/ElasticSearchParticipantDto.java

```diff
@@ -22,6 +22,7 @@
     private String status;
     private ESDsm dsm;
     private String ddp;
+    private List<String> governedUsers;
```

Comment:
`governedUsers` is the exact name how it comes from ElasticSearch? asking because converting source map of ElasticSearch to the object of `ElasticSearchParticipantDto` is done by using `GSON` and field names needs to match names which come from ElasticSearch's source map

Reply:
yes, that is the ES object name